pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -780,6 +780,7 @@ func (p *Pod) cleanupExcessPods(ctx context.Context, c client.Client, totalCount
 				p.excessPodExpectations.ObservedUID(log, p.key, pod.UID)
 				return err
 			}
+			r.Event(&pod, corev1.EventTypeNormal, jobframework.ReasonStopped, "Excess pod deleted")
```

Comment:
Maybe we should use a different Reason here, as we are not "stopping" the job.

Reply:
I see, this is different than preemption, because in case of preemption we stop the Job, represented as a group of pods, meaning we stop all pods. Here we only delete a subset of excess pods, but we are not stopping the job as a whole.

Maybe just `Reason: DeletedPod`?